upgrade/helpers/tasks.py

```diff
@@ -998,3 +998,21 @@ def upgrade_validation(upgrade_type=False):
     if upgrade_type:
         run('hammer ping', warn_only=True)
     run('katello-service status', warn_only=True)
+
+
+def mongo_db_engine_upgrade(upgrade_type):
+    """
+    The purpose of this method to perform the upgrade of mongo DB database engine
+    from MMAPv1 to WiredTiger.
+    :param str upgrade_type: upgrade type would be a string, depends on choice If user
```

Comment:
Remove this from docstring 'upgrade type would be a string' as :param str is already given.

Instead of upgrade_type string parameter. If you want to choose between satellite and capsule, better u go for default parameter as satellite=True else False for capsule.

Reply:
>Instead of upgrade_type string parameter. If you want to choose between satellite and capsule, better u go for default parameter as satellite=True else False for capsule.

Is it right to put an extra check for validation whether it is Satellite or Capsule?